added hts Freiburg workshop event in Summer

teresa-m opened the pull request:
@Sch-Da: here is the event page for the summer Freiburg Workshop.
I added the registration link and send the deadline to the 20th of May. If this does not fit, I can update it.

Sch-Da commented on events/2025-06-30-hts-workshop-freiburg.md:
```diff
@@ -0,0 +1,135 @@
+---
+layout: event
+title: "Workshop on high-throughput sequencing data analysis with Galaxy"
+# or: From Data to Discovery: Metagenomics, RNA-Seq - NGS Bioinformatics with Galaxy
```
I lean a bit more towards the
"From Data to Discovery" to show the difference to the regular Galaxy workshop we offer. But you all have more experience.

teresa-m replied:
I also like the other one more. It is just that this is the 'regular' name. So maybe even better to rename, as we will offer a bit of a different content?

Sch-Da replied:
Thought so, too. But might make it less easy to track - on the downside. Good reasons for either

Sch-Da commented on events/2025-06-30-hts-workshop-freiburg.md:
```diff
@@ -0,0 +1,135 @@
+
+contributions:
+  organisers: [Sch-Da, teresa-m]
+  instructors: [pavanvidem, teresa-m, dianichj, EngyNasr, paulzierep]
```
Maybe add Mina as well?

teresa-m replied:
Mina is not on the GTN contributors.yaml, I will add her. (Just was to lazy before)

Sch-Da replied:
Thanks a lot!

Sch-Da commented on events/2025-06-30-hts-workshop-freiburg.md:
```diff
@@ -0,0 +1,135 @@
+
+registration:
+  link: https://forms.gle/AQ7n7zi1rUd995Me9
+  deadline: 2025-05-20
```
```suggestion
  deadline: 2025-05-11
```
We only have around 10 spots so I think this will actually be booked rather quickly. Therefore, t not have people's hopes up, i would set the deadline a little earlir

Sch-Da commented on events/2025-06-30-hts-workshop-freiburg.md:
```diff
@@ -0,0 +1,135 @@
+        time: "16:00"
+
+  - section: "Tuesday: RNA Sequencing"
+    description: We will do at least one coffee break in the morning, one in the afternoon, and 1h lunch break around noon.
```
```suggestion
    description: We will do at least one coffee break in the morning, one in the afternoon, and a one-hour lunch break around noon.
```

Sch-Da commented on events/2025-06-30-hts-workshop-freiburg.md:
```diff
@@ -0,0 +1,135 @@
+        time: "09:15-17:00"
+
+  - section: "Wednesday: Single-Cell Sequencing"
+    description: We will do at least one coffee break in the morning, one in the afternoon, and 1h lunch break around noon.
```
```suggestion
    description: We will do at least one coffee break in the morning, one in the afternoon, and a one-hour lunch break around noon.
```

Sch-Da commented on events/2025-06-30-hts-workshop-freiburg.md:
```diff
@@ -0,0 +1,135 @@
+
+
+  - section: "Thursday: Metagenomics"
+    description:  We will do at least one coffee break in the morning, one in the afternoon, and 1h lunch break around noon.
```
```suggestion
    description:  We will do at least one coffee break in the morning, one in the afternoon, and a one-hour lunch break around noon.
```

Sch-Da commented on events/2025-06-30-hts-workshop-freiburg.md:
```diff
@@ -0,0 +1,135 @@
+        time: "17:00"
+
+  - section: "Friday: Metagenomics"
+    description:  We will do at least one coffee break in the morning, one in the afternoon, and 1h lunch break around noon.
```
```suggestion
    description:  We will do at least one coffee break in the morning, one in the afternoon, and a one-hour lunch break around noon.
```

Sch-Da reviewed (commented):
Thanks a lot for setting this up @teresa-m. It looks great. I added some minor comments or questions

teresa-m commented on events/2025-06-30-hts-workshop-freiburg.md:
```diff
@@ -0,0 +1,135 @@
+# Welcome to the Comprehensive Galaxy Workshop: From Introduction to Advanced Applications
+
+Embark on a deep dive into the world of Galaxy, the leading platform for data-intensive biomedical research. This workshop is designed for researchers, students, and data analysts who wish to
+harness the full potential of Galaxy in transcriptomic, single-cell and metagenomics studies. Whether you're a beginner seeking to understand the basics or an intermediate user looking to refine your skills in specific applications,
```
Yes I already included and excluded the topics not taught and now not taught.

Sch-Da reviewed (approved):
Then I think we are set, right?

teresa-m commented:
@Sch-Da, sorry I was too tired yesterday. I will do the updates we discussed yesterday today and then we are good to go.

teresa-m commented:
I updated the Title and added Mina as a trainer and to the contributor yaml.
@Minamehr, if you want, you can check that the info in the contributors.yaml is correct and feel free to add more info if you want.

Sch-Da commented on events/2025-06-30-hts-workshop-freiburg.md:
```diff
@@ -0,0 +1,134 @@
+
+## Registration
+
+The number of places in this workshop is limited. The course takes place only in person (not online, not hybrid). There are no fees for the workshop but you need to care for your accommodation, travel costs and catering by yourself.
```
```suggestion
The number of places in this workshop is limited. The course takes place only in person (not online, not hybrid). There are no fees for the workshop but you need to care for your accommodation, travel costs and catering yourself.
```

Sch-Da commented on events/2025-06-30-hts-workshop-freiburg.md:
```diff
@@ -0,0 +1,134 @@
+# Add GTN tutorials by supplying the topic and tutorial name
+program:
+  - section: "Monday: Introduction and Quality Control"  # section title is optional
+    description: We will do at least one coffee break in the morning, one in the afternoon, and 1h lunch break around noon.
```
```suggestion
    description: We will do at least one coffee break in the morning, one in the afternoon, and a one-hour lunch break around noon.
```

teresa-m replied:
Thanks a lot for checking again. If you are fine with it now we can merge.

Sch-Da reviewed (commented):
Thanks a lot. Sorry, I found two minor things. Otherwise, it looks great to me :)

Sch-Da reviewed (approved):
thanks so much @teresa-m
